### ISSUE TITLE
Create a chat room and join a room are missing functionalities

### ISSUE THREAD

**Issue by arranger1044** [arranger1044/iSoul#6]
In the Chat room List Window there are two blank toolbar item for creating a chat room and joining an already existing one but no functionality is linked to them in the code.
A couple of icons would be necessary : )

**Comment by arranger1044** [comment]
I created the glue code to create/join a chat room from the UI. Icons still missing, any designer around? : )

**Comment by djbe** [comment]
No, but I can give it a try. Do you have anything in mind?

**Comment by arranger1044** [comment]
maybe for the join chat room icon we can have a balloon (representing a chat room) with an arrow pointing to it (representing the act to enter), and for the create chat room icon a balloon with a plus sign in a corner.

**Comment by djbe** [comment]
In case you're wondering, I'll create them after wednesday (when my exams finish).

**Comment by arranger1044** [comment]
There is no need to hurry David, real life always comes first! Moreover, you are already helping this project, I wish we were more people

**Comment by djbe** [comment]
I created one with a "+" in the corner, but that doesn't seem to fit IMO. If we look at the current icon set, to create a private chat, it's just a ballon (no +).

I've posted the icons here: http://imgur.com/a/N6F8h (newer iterations will also appear there).

I think the arrow needs a bit of adjusting, but what do you think? In essence, it's the same as the current chatroom toolbar icon (but bigger).

**Comment by drvid** [comment]
Hi I am an OS X app designer and developer wanting to contribute to this project and learn more Obj-C ... I have been involved in small ways since 2008. Please read about my experience with iSoul in this post on my site http://wp.me/p1XOf8-7m

**Comment by arranger1044** [comment]
You are right about creating a private chat icon, i suggest we can change it too (the simple chat icon is for...chat rooms), could you post the one with the plus?.
About the arrow, what about moving it a bit on the right, coming from outside the balloon and entering it?

**Comment by arranger1044** [comment]
Hi drvid, your site seems unreachable for me atm, could you please use our newsletter or open another thread somewhere?

**Comment by djbe** [comment]
I was able to reach it somewhat using google cache:
http://webcache.googleusercontent.com/search?q=cache:kyB5c23GTG0J:drvid.ca/words/2012/01/isoul-brings-soulseek-to-osx/+drvid+isoul&hl=en&client=safari&strip=1

I've added some updated icons to the gallery

**Comment by drvid** [comment]
Sorry my bad. I am kind of new to GitHub, where do you guys chat?
The link should work now but yeah the jist of it is in that cached version.

**Comment by djbe** [comment]
most of the discussions happen here, although I also created a google group for people who prefer to use that:
http://groups.google.com/group/isoul-support

**Comment by drvid** [comment]
I was thinking something a little more realtime and less public/permanent. Actually why don't we all just lurk and discuss in the #iSoul channel on Soulseek considering most of us are already there? Add me to your friends list in slsk, it makes the most sense to me since we can also talk to the other users in there too.

**Comment by djbe** [comment]
not now for me, I'm going to sleep now (trip to Spain tomorrow morning = in a few hours).

**Comment by arranger1044** [comment]
David, how about making the plus and the arrow bigger? Here is what I mean http://desmond.imageshack.us/Himg830/scaled.php?server=830&filename=fotodel030212alle18112.jpg&res=medium, moreover we could make those artifacts the same color of the background balloon (with the same reflection style maybe?)
By the way, at the moment simple icons for the autojoin, create chat, join chat, reload list are fine, whichever they are

**Comment by djbe** [comment]
Errr, to clarify: you want the plus and the arrow to be blue (gradient)? With a black border or without?

**Comment by arranger1044** [comment]
It is just some proposal David : D, if we had a community we could make a poll. I'd like to see those without border and with the overlay effect. Thanks for your efforts, how did your exams go?

**Comment by djbe** [comment]
Dunno, okay I think, still waiting for the results (should be another week or more).

Uploaded some new versions, am I going in the right direction? bigger artifacts? changing gradient position/direction?

**Comment by arranger1044** [comment]
The arrow seems fine to me, the plus sign shall be bigger instead (more or less the same size of the arrow) 
Il giorno 04/feb/2012, alle ore 21:35, David Jennes ha scritto:

**Comment by djbe** [comment]
done

**Comment by arranger1044** [comment]
I mean, each part of the cross shall be thicker, like in the image I uploaded (I hope I'm not going to make you crazy)
Il giorno 04/feb/2012, alle ore 21:43, David Jennes ha scritto:

**Comment by djbe** [comment]
That's as thick as it gets with normal fonts I think (also added a version with an ever bigger plus).

offtopic: I wonder why imgur creates such shitty thumbs for transparent pngs...

**Comment by arranger1044** [comment]
Can't you create it as you created the arrow? using a font makes the edges thinner
Btw, I'm thinking that maybe those images are a bit smaller for a toolbar item, well they'll be fine for now

Il giorno 04/feb/2012, alle ore 22:06, David Jennes ha scritto:

**Comment by djbe** [comment]
Actually the arrow is a special text symbol (using some font variation). Why make it difficult when it can be done easily :-D
Anyway, new plus uploaded, I found a variation of the plus symbol with shorter (and thicker) bars.

**Comment by arranger1044** [comment]
I guess we're done David, good job!

Il giorno 05/feb/2012, alle ore 02:31, David Jennes ha scritto:

**Comment by djbe** [comment]
Thx :-D

So, we're going with the last 2 right?
![Create](http://i.imgur.com/zE2lg.png)
![Join](http://i.imgur.com/EvT6f.png)

**Comment by arranger1044** [comment]
Yep, I guess we can add a 'recycle-reload' double circular arrow for the reload chat list button, but for the autojoin?
Il giorno 05/feb/2012, alle ore 14:41, David Jennes ha scritto:

**Comment by djbe** [comment]
Wait, what?

**Comment by drvid** [comment]
My 2 cents: They look nice, I'd suggest the arrow and the plus symbol sit fully inside the white chat bubble, centered. This would be easier to read at a glance. If you need an icon for 'list chatrooms' you could follow suit with the 3 horizontal lines symbol (i forget it's name) inside the bubble.

**Comment by djbe** [comment]
done, although the create icon looks a bit weird...

**Comment by arranger1044** [comment]
I do not like them, I think the third symbol in the icon (the plus or the the arrow) shall be aligned and spaced like the previous 2 i.e. put on the right a bit off the previous one and a bit down (it would be nicer to have all three perfectly aligned and spaced)
Il giorno 06/feb/2012, alle ore 16:22, David Jennes ha scritto:

**Comment by drvid** [comment]
I like this idea it does look better I think. But you need to shrink the plus symbol inside 'create' by about ~5% to equalize the whitespace padding around the symbols inside the bubble. And one side of the gradient looks vague so either less blue gradient or perhaps a blue inner shadow on those symbols to make them "pop" with more contrast... thicker chat list lines too, try to balance the amount of blue and white in there. These are only suggestions.

**Comment by djbe** [comment]
First decide wether the symbols should be on the border of the bubble (like @arranger1044 says) or in the middle (like @drvid says). I'm not going to do everything double :-D

**Comment by drvid** [comment]
Hi @djbe please upload your PSD or layered sources somewhere (github?) and I can create the icons to show you guys what I mean. BTW at this point I would say the actual functionality is more important that the icon chosen to represent it.

**Comment by arranger1044** [comment]
@drvid I guess you could do it by yourself starting from the icons included in iSoul.
@djbe the only problem I see is related to the enlargement of a small icon, it gets a bit raw
